unit/Makefile

```diff
@@ -13,47 +13,26 @@ SRC += unit_tests.cpp \
        analyses/does_remove_const/does_type_preserve_const_correctness.cpp \
        analyses/does_remove_const/is_type_at_least_as_const_as.cpp \
        goto-programs/goto_trace_output.cpp \
-       goto-programs/class_hierarchy_output.cpp \
-       goto-programs/class_hierarchy_graph.cpp \
-       goto-programs/remove_virtual_functions_without_fallback.cpp \
```

Comment:
If those three depend on Java then the implementation ought to move to the `java_bytecode` folder.